task/control-plane-agent/src/inventory.rs

```diff
@@ -168,6 +168,15 @@ const OUR_DEVICES: &[DeviceDescription<'static>] = &[
         capabilities: DeviceCapabilities::UPDATEABLE,
         presence: DevicePresence::Present, // TODO: ok to assume always present?
     },
+    // If we're building for sidecar, we always claim to have a VSC7448.
+    #[cfg(feature = "sidecar")]
+    DeviceDescription {
+        component: SpComponent::VSC7448,
+        device: SpComponent::VSC7448.const_as_str(),
+        description: "Management network switch",
+        capabilities: DeviceCapabilities::HAS_MEASUREMENT_CHANNELS,
+        presence: DevicePresence::Present, // TODO: ok to assume always present?
```

Comment:
Kinda – it takes ~4 seconds to boot, but also, while booting, we can't access the management network 😂